themes/src/main/resources/theme/base/admin/resources/js/controllers/realm.js

```diff
@@ -1039,6 +1037,7 @@ module.controller('RealmIdentityProviderCtrl', function($scope, $filter, $upload
         $scope.identityProvider.firstBrokerLoginFlowAlias = 'first broker login';
         $scope.identityProvider.config.useJwksUrl = 'true';
         $scope.identityProvider.config.syncMode = 'IMPORT';
+        $scope.identityProvider.config.allowCreate = 'true';
```

Comment:
This should be _at the end of `initSamlProvider` function_, i.e. line 1013.

Reply:
I have one question based on this change. 
I put default value only for create. 
When IdP was updated, we do not want to change allowCreate value. However, when upgrading from older version allowCreate  will not be in databse. So, when admin user try to update SAML Identity Provider value will be set to false. Is this a problem?
We do not have problem, when user is trying to login ( default to true).